lepidopter-fh/etc/cron.daily/remove_old_logs

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-# This cronjob has been created to overcome a limitation of ooniprobe log
-# rotations. It deletes files modified by the pattern 'ooniprobe.log.*_*' the
-# last 6 days for the ${OONI_LOGS} directory
-source /etc/ooniprobe/ooniconfig.sh
-
-find ${OONI_LOGS} -name "ooniprobe.log.*_*" -mtime +6 -type f -delete
```

Comment:
As we discussed with @hellais it makes more sense to remove old log files since ooniprobe doesn't remove log files after log rotation.